core/system/src/mpu/vmpu.c

```diff
@@ -95,6 +95,10 @@ static int vmpu_sanity_checks(void)
     assert( (uint32_t) __uvisor_config.bss_main_end == (SRAM_OFFSET_START +
                                                         UVISOR_SRAM_LENGTH) );
 
+    /* Verify OS private BSS */
+    assert(__uvisor_config.os_priv_start == __uvisor_config.bss_main_end);
+    assert(__uvisor_config.os_priv_end > __uvisor_config.bss_main_start);
```

Comment:
My second assertion was checking that the OS private BSS ends after the start of uVisor main BSS, which should be strictly greater than, as we always reserve a non-zero UVISOR_SRAM_SIZE amount of data in the main BSS.

Your proposed second assertion is another good one to add, though.